telcell/data/parsers.py

```diff
@@ -0,0 +1,53 @@
+import csv
+from itertools import groupby
+from pathlib import Path
+from typing import List, Union
+
+from telcell.data.models import Measurement, Track
+
+
+def parse_measurements_csv(path: Union[str, Path]) -> List[Track]:
+    """
+    Parse a `measurements.csv` file into `Track`s. The following columns are
+    expected to be present:
+
+        - track
+        - sensor
+        - celldb.wgs84.lat
+        - celldb.wgs84.lon
+        - timestamp
+
+    Any additional columns are stored under the `extra` attribute of each
+    resulting `Measurement` object.
+
+    :param path: The path to the `measurements.csv` file that should be parsed
+    :return: All `Track`s that were constructed from the data
+    """
+    tracks = []
+    with open(path, "r") as f:
+        reader = csv.DictReader(f)
+        # We assume the rows in the csv file are already sorted by:
+        #   1. track (person)
+        #   2. sensor (device)
+        #   3. timestamp
+        # If this is not the case, we first have to call `sorted()` here.
```

Comment:
Seems like an useful addition to just sort anyway if it is important. 

`reader = sorted(reader, key=lambda d: (d["track"], d["sensor"],  d["timestamp"]))`

Reply:
Maybe not exactly with this. It sorts, but somehow the measurement lists decreases with 1000 records